api/tests/mocha/controllers/changes.spec.js

```diff
@@ -25,7 +26,7 @@ describe('Changes controller', () => {
   it('should respond with ddoc and service worker meta', async () => {
     sinon.stub(db.medic, 'changes').resolves({
       changes: [
-        { id: 'service-worker-meta' },
+        { id: DOC_IDS.SERVICE_WORKER_META },
         { id: '_design/medic-client', },
         { id: 'org.couchdb.user:user', },
         { id: 'settings' },
```

Comment:
Are these three strings candidates for constants?

`'org.couchdb.user:user'` only appears in tests. `'_design/medic-client'` appears in other files, not just tests.